pkgs/applications/networking/sniffers/wireshark/cmake.patch

```diff
@@ -0,0 +1,19 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -19,6 +19,7 @@
+ # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ #
+ 
++cmake_minimum_required(VERSION 3.7)
```

Comment:
Can be merged as it is, but could you also try to send the patch upstream. This makes future updates easier.

Reply:
Does this looks interesting too you? @alagoutte
It fixes compilation in nixpkgs on macOS for us.

Reply:
the problem here is that the minimum version must be defined before `project`. Otherwise it complains about clang.

Reply:
Hi,

Yes, you can propose the fix on upstream